dbus/fanatec_input.py

```diff
@@ -7,33 +7,41 @@
 from pydbus import SystemBus
 from gi.repository import GLib
 
-FANATEC_VENDOR_ID='0EB7'
-CSL_ELITE_PEDALS_DEVICE_ID='6204'
-CSL_ELITE_PS4_WHEELBASE_DEVICE_ID='0005'
+FANATEC_VENDOR_ID = "0EB7"
+CSL_ELITE_PEDALS_DEVICE_ID = "6204"
+CSL_ELITE_PS4_WHEELBASE_DEVICE_ID = "0005"
+CSL_DD_P1_V2_WHEEL_ID = "0020"
```

Comment:
Rename to `CSL_DD_WHEELBASE_ID` (so that it's similar to what's in the drivers `hid-ftec.h`). This PID only specifies the wheelbase, it does not give information on the mounted wheel.
(This information can be obtained from the `wheel_id` sysfs file, though...)

Reply:
So this problem is also related to some other of your requested changes, I will go into detail elsewhere.